### PR TITLE
Update deps/changelog

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: etc/ctrl.yml [hunk 1]
```diff
@@ -134,3 +134,66 @@ mgmt:
   #influxdb:
     #url:                http://localhost:8086
     #database:           ziti
+    #
+    
+
+# web - optional
```
Comment (dovholuknf) [inline etc/ctrl.yml:140]: was this file just never updated with the new stuff, right?

Comment (plorenz) [reply etc/ctrl.yml:140]: yes, and since it will become required soon with the fabric rest API, I though I should add it in

Comment (dovholuknf) [reply etc/ctrl.yml:140]: oh - this is the "no edge" config file... maybe this file should just be deleted or the other one should be mv'ed onto this file name?

Comment (plorenz) [reply etc/ctrl.yml:140]: I think it makes sense to have both. There's maybe an argument that with edge is the most common case, so edge.yml should be with edge and we could have a ctrl.fabric.only.yml.

Comment (plorenz) [reply etc/ctrl.yml:140]: I don't think it should matter much to people who aren't developing ziti though. It should be easy to create your own config easily.

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**